src/components/NumberInput/index.js

```diff
@@ -0,0 +1,77 @@
+import { Lightning } from 'wpe-lightning-sdk'
+import Config from './config.js'
+import * as player from '@/services/player/'
+import { channelsServiceInit, getCurrentProgram, getChannel } from '@/services/ChannelsService'
+import { ChannelNumber } from './channelnumber.js'
+
```

Comment:
Add:
import theme from '@/themes/default'

Reply:
import theme from '@/themes/default' added in  NumberInput/index.js